### PR TITLE
NO-JIRA: E2E: Adjust tests for cgroupv2 changes

### PR DESCRIPTION
**Pull request by mrniranjan**

### CODE REVIEW

**Comment by mrniranjan** [comment]
This PR is dependent on #991

**Review by ffromani — COMMENTED** [review]
/approve

mostly LGTM but few questions inside

/cc @Tal-or

File: test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go [hunk 1]
```diff
@@ -84,9 +82,6 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 				By("Modifying profile")
 				profile.Spec.WorkloadHints = nil
 
-				if *profile.Spec.RealTimeKernel.Enabled == false {
-					testutils.KnownIssueJira("OCPBUGS-28828")
-				}
```
[the remark this answers is not in the document]
Comment (mrniranjan) [reply test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go:89]: This is not related to cgroupv2 , it's a bug which was closed as not a bug . I will probably move this to another commit.

File: test/e2e/performanceprofile/functests/7_performance_kubelet_node/kubelet.go [hunk 2]
```diff
@@ -222,6 +221,38 @@ var _ = Describe("[ref_id: 45487][performance]additional kubelet arguments", Ord
...
+
+				By("Waiting when mcp finishes updates")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+			}
+		})
 
 	})
 })
+
+func updateAnnotation(profileAnnotations map[string]string, annotations string) map[string]string {
```
[the remark this answers is not in the document]
Comment (mrniranjan) [reply test/e2e/performanceprofile/functests/7_performance_kubelet_node/kubelet.go:250]: ack

**Pull request merged**